src/features/common/Layout/Header/accountHeader.tsx

```diff
@@ -0,0 +1,84 @@
+import React,{ useEffect } from 'react';
+import styles from './Header.module.scss';
+import { useRouter } from 'next/router';
+
+import Settings from '../../../../../public/assets/images/icons/userProfileIcons/Settings';
+import SettingsModal from '../../../user/UserProfile/components/SettingsModal';
+import { useAuth0 } from '@auth0/auth0-react';
+import BackButton from '../../../../../public/assets/images/icons/BackButton';
+import i18next from '../../../../../i18n'
+import {UserPropsContext} from '../UserPropsContext';
+const { useTranslation } = i18next;
+
+export default function AcountHeader(props: any) {
+  const {userprofile} = React.useContext(UserPropsContext);
+  const [forceReload, changeForceReload] = React.useState(false);
+  const { t } = useTranslation(['me']);
+
+  const {
+    isLoading,
+    isAuthenticated,
+    loginWithRedirect,
+    getAccessTokenSilently,
+    logout
+  } = useAuth0();
+  const router = useRouter();
+
+  const [settingsModalOpen, setSettingsModalOpen] = React.useState(false);
+  const handleSettingsModalClose = () => {
+    setSettingsModalOpen(false);
+  };
+  const handleSettingsModalOpen = () => {
+    setSettingsModalOpen(!settingsModalOpen);
+  };
+  // editProfile modal  (from settings modal)
+  const [editProfileModalOpen, setEditProfileModalOpen] = React.useState(false);
+  const handleEditProfileModalClose = () => {
+    setEditProfileModalOpen(false);
+  };
+  const handleEditProfileModalOpen = () => {
+    setEditProfileModalOpen(true);
+  };
+    return (
+      <div>
+      <div className={styles.headerBG}>
+        <div className={styles.accountsHeader}>
+          <div className={styles.navContainer}>
+            <button
+              onClick={() => {
+                router.back();
```

Comment:
Going backwards the browser history can result into an error if going through the Auth0 login process for URLs like http://localhost:3000/manage-projects/add-project or http://localhost:3000/account/history as following:

<img width="754" alt="Bildschirmfoto 2021-04-30 um 16 12 18" src="https://user-images.githubusercontent.com/1532418/116707355-deb9ae80-a9ce-11eb-8f7d-64c277442690.png">

Reply:
This should be solved in https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1057